MIGRATION_GUIDE.md

```diff
@@ -4,6 +4,33 @@ This document is meant to help you migrate your Terraform config to the new newe
 describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behavior
 across different versions.
 
+## v0.95.0 ➞ v0.96.0
+
+### *(breaking change)* resource_monitor resource
+Removed fields:
+- `set_for_account` (will be settable on account resource, right now, the preferred way is to set it through unsafe_execute resource)
+- `warehouses` (can be set on warehouse resource, optionally through unsafe_execute resource)
+- `suspend_triggers` (now, `suspend_trigger` should be used)
```

Comment:
No, those were the previous fields (https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/resource_monitor#optional) we had `suspend_triggers` and `suspend_trigger`, but the singular version only made sense, because we can only specify one suspend trigger (additionally the plural one was deprecated).